Bindings/Python/__init__.py

```diff
@@ -1,9 +1,9 @@
-from simbody import *
-from common import *
-from simulation import *
-from actuators import *
-from analyses import *
-from tools import *
-from examplecomponents import *
+from .simbody import *
+from .common import *
+from .simulation import *
+from .actuators import *
+from .analyses import *
+from .tools import *
+from .examplecomponents import *
```

Comment:
Yes. I tested this syntax locally for python 3, and I tested it on python 2 using travis-ci/appveyor (both of which run the python tests with python 2).

There's some discussion about this in the SWIG docs (http://www.swig.org/Doc3.0/Python.html#Python_absrelimports; see subsection 36.11.4, "Importing from \_\_init\_\_.py")